feature-libs/product-configurator/rulebased/root/interceptor/cpq-configurator-rest.interceptor.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  HttpErrorResponse,
+  HttpEvent,
+  HttpHandler,
+  HttpInterceptor,
+  HttpRequest,
+  HttpResponse,
+} from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError } from 'rxjs';
+import { catchError, switchMap, take, tap } from 'rxjs/operators';
+
+import { CpqAccessData } from './cpq-access-data.models';
+import { CpqAccessStorageService } from './cpq-access-storage.service';
+
+const HEADER_ATTR_CPQ_SESSION_ID = 'x-cpq-session-id';
+const HEADER_ATTR_CPQ_NO_COOKIES = 'x-cpq-disable-cookies';
```

Comment:
Let's put those consts as `protected` `readonly` propetires of the class, so customers can customize them if needed.

Reply:
Pick!

Reply:
Done!